torchao/quantization/__init__.py

```diff
@@ -56,6 +58,8 @@
     Int8DynamicActivationInt4WeightConfig,
     Int8DynamicActivationInt8WeightConfig,
     Int8WeightOnlyConfig,
+    PlainLayout,
```

Comment:
is this exposed in public API / on HF / etc?

I don't think this abstraction is clean and I don't want to expose it, this is not really a "layout" rather than a "layout with some other arguments mixed in for convenience".  We should talk about what to do with this before exposing it.